packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -243,10 +246,11 @@ class Navigator extends StatefulWidget {
     return Navigator.of(context).pop(result);
   }
 
-  /// Calls pop() repeatedly until the given route is the current route.
-  /// If it is already the current route, nothing happens.
-  static void popUntil(BuildContext context, Route<dynamic> targetRoute) {
-    Navigator.of(context).popUntil(targetRoute);
+  /// Calls [pop()] repeatedly until the predicate returns false.
+  /// The predicate may be applied to the same route more than once if
+  /// [Route.willHandlePopupInternally] is true.
```

Comment:
I've noticed that I have trouble typing just "pop". It's like a nervous tick.  Thanks for catching that.